curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5efada803cbd2bbdab94e332.md

```diff
@@ -35,7 +35,7 @@ You should have a third `img` element nested in the `figure` element.
 const catsImg = document.querySelectorAll('figure > img')[1];
 assert(
   catsImg &&
-    catsImg.getAttribute('src').toLowerCase() === 'https://cdn.freecodecamp.org/curriculum/cat-photo-app/cats.jpg'
+    catsImg.getAttribute('src')
```

Comment:
I would just check it like this:
`assert(catsImg)`

That pretty much just checks for exactly what the hint says. The hint doesn't say the `src` is required, so you could add `<img>` and it wouldn't pass, even though you added the element.

Reply:
Thanks for reviewing my pr! I just changed the code to only use `assert(catsImg)`